ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1884,6 +1884,9 @@ tests:
       AUX_HOST: openshift-qe-metal-ci.arm.eng.rdu2.redhat.com
       RESERVE_BOOTSTRAP: "false"
       architecture: amd64
+    observers:
```

Comment:
Can we add this at workflow/chain level or only here?

Reply:
testing it

Reply:
tested, it works only there